### DISCUSSION TITLE
Reset opendcim password

### DISCUSSION THREAD

**Discussion by Prody33** [opendcim/openDCIM#1473] [Q&A]
Hello,

We are currently using LDAP authentication and we are going to make some changes.
To be on the safe side, I tried to log on to the local admin account "opendcim", but unfortunately it no longer works.
We have tried to reinitialise it but it is impossible to find the procedure or even the password in the database. 

`
+----------+---------------+-----------------+------------+-------------+--------------+--------------+-------------+-----------+-----------+----------+
| UserID   | Name          | AdminOwnDevices | ReadAccess | WriteAccess | DeleteAccess | ContactAdmin | RackRequest | RackAdmin | SiteAdmin | Disabled |
+----------+---------------+-----------------+------------+-------------+--------------+--------------+-------------+-----------+-----------+----------+
| opendcim | Default Admin |               0 |          1 |           1 |            1 |            1 |           1 |         1 |         1 |        0 |
+----------+---------------+-----------------+------------+-------------+--------------+--------------+-------------+-----------+-----------+----------+
`
I've tried searching everywhere from the interface or even the files without success, I can't find anything.

I haven't found any information about a procedure, either in the documentation, or in the wiki, or on the net.

Could you please help me?

Thanks you for Advance.

**Comment by wilpig** [comment]
If you are configured for ldap auth you cannot use any local accounts.  If you want to use a mix of local accounts and ldap then you need to use simple auth in your apache config.  The only thing you will find in the database is the username and permissions associated with that username, there are no passwords since opendcim does not handle authentication.

**Reply by Prody33** [reply]
Thanks for your feedback Wilpig,
I understand better. So I've added the following directive

`     <Directory /var/www/dcim>
         AllowOverride All
         AuthType Basic
         AuthName "openDCIM"
         AuthUserFile /var/www/dcim/.htpasswd
         Require valid-user
     </Directory>`

Then generate an .htaccess file in the indicated folder
opendcim:$apr1$************************vxCzj1

But I still can't log in to this account. A popup window opens in which I enter the users and password several times but without success.

Am I missing something?

Thanks for your help

**Reply by Prody33** [reply]
In fact, since this option was introduced I simply can't connect at all.
Neither in AD nor in Local. What's more, I've also noticed that I don't even have the authetification page any more. It's this famous popup window that appears immediately before I even have the site interface.

I'm sure that before we had both and that we didn't have this popup window. It's possible that someone has deactivated it.

At the moment I need to change my AD and I can't take the risk of doing that without being able to authenticate myself by another route.

**Reply by Prody33** [reply]
Another very surprising point.

I've created a new user from the interface, and I can see him persisting on the site, but he simply doesn't exist in the database.
[Image New User](https://ibb.co/4RZX32J)
[Image BDD](https://ibb.co/x557S8d)

How is this possible?

**Comment by wilpig** [comment]
You didn't switch the authentication back in the db.inc.php

```
    //define( "AUTHENTICATION", "Apache" );
    define( "AUTHENTICATION", "LDAP" );
```
